src/yuzu/configuration/configure_hotkeys.cpp

```diff
@@ -82,24 +91,28 @@ void ConfigureHotkeys::Configure(QModelIndex index) {
         return;
     }
 
-    if (IsUsedKey(key_sequence) && key_sequence != QKeySequence(previous_key.toString())) {
-        QMessageBox::warning(this, tr("Conflicting Key Sequence"),
-                             tr("The entered key sequence is already assigned to another hotkey."));
+    if (IsUsedKey(key_sequence, used_action) &&
+        key_sequence != QKeySequence(previous_key.toString())) {
+
+        QMessageBox::warning(
+            this, tr("Conflicting Key Sequence"),
+            tr("The entered key sequence is already assigned to: %1").arg(used_action));
     } else {
         model->setData(index, key_sequence.toString(QKeySequence::NativeText));
     }
 }
 
-bool ConfigureHotkeys::IsUsedKey(QKeySequence key_sequence) const {
-    for (int r = 0; r < model->rowCount(); r++) {
+bool ConfigureHotkeys::IsUsedKey(QKeySequence key_sequence, QString& used_action) const {
```

Comment:
```suggestion
std::pair<bool, QString> ConfigureHotkeys::IsUsedKey(QKeySequence key_sequence) const {
```

Then calling sites can just do:

```cpp
const auto [is_used, used_action] = IsUsedKey(sequence);
```

Instead of needing to construct the string separately before calling the function.